modules/virt-example-snippets-nmstate-IP-management.adoc

```diff
@@ -0,0 +1,132 @@
+// Module included in the following assemblies:
+//
+// * virt/node_network/virt-updating-node-network-config.adoc
+
+[id="virt-example-snippets-nmstate-IP-management_{context}"]
+= Example snippets: IP management
+
+The following are example configuration snippets for different methods of IP management. 
+
+These snippets use the `ethernet` interface type to simplify the example while showing the related context in the Policy configuration. These IP management examples can be used with the other interface types. 
+
+== Static
+
+The following snippet statically configures an IP address on the ethernet interface:
+
+[source,yaml]
+----
+...
+    interfaces:
+    - name: eth1
+      description: static IP on eth1
+      type: ethernet
+      state: up
+      ipv4:
+        address:
+        - ip: 192.168.122.250 <1>
+          prefix-length: 24
+        enabled: true
+...
+----
+<1> Replace with the static IP address for the interface.
+
+== No IP address
```

Comment:
⚓  (and the same applies to other headings)

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings

Reply:
Aye aye!